CMakeLists.txt

```diff
@@ -148,6 +148,7 @@ find_package(ICU COMPONENTS uc i18n REQUIRED)
 find_package(fmt 10.0.0 REQUIRED)
 find_package(libdeflate REQUIRED)
 find_package(libxcrypt REQUIRED)
+find_package(p11-kit REQUIRED)
```

Comment:
instead of using "p11" as the prefix of the title of the commit message, could you please use "main" instead? as this change is more related to 

1. preparing the building system to detect p11-kit
2. override p11-kit settings early in main

IMHO, the touched areas are "build" and "main", "p11" is the library we are using.

Reply:
done